mllaunchpad/wsgi.py

```diff
@@ -26,27 +26,20 @@
 try:
     conf = config.get_validated_config()
 except FileNotFoundError:
-    logger.error(
-        "Config file could not be loaded. Starting the Flask application "
-        "will fail."
-    )
+    conf = None
```

Comment:
(Hint: If you are okay with the suggestions I make, you can accept them directly using the buttons below the suggestion in the GitHub web UI.)

My comment on this part: Please leave this logging in, we need it to help the user if there **really** is a problem loading the configuration. Click "commit suggestion" if okay for you.
```suggestion
    logger.error(
        "Config file could not be loaded. Starting the Flask application "
        "will fail."
    )
    conf = None
```